packages/arb-token-bridge-ui/src/token-bridge-sdk/hooks/__tests__/useBalance.test.tsx

```diff
@@ -11,7 +11,7 @@ import { PropsWithChildren } from 'react'
 import { MultiCaller } from '@arbitrum/sdk'
 
 // Create a new cache for every test
-const Container = ({ children }: PropsWithChildren<{}>) => (
+const Container = ({ children }: PropsWithChildren<any>) => (
```

Comment:
why is the type changed?

Reply:
As soon as the SDK folder was imported inside the UI, the next-eslint rules applied to it, highlighting this type {} to be invalid. 
<img width="400" alt="image" src="https://user-images.githubusercontent.com/7558499/235096522-f0ae7c0d-5af9-4bc6-a1e1-50ef9c066e8d.png">

As Bartek suggested, we are now using unknown instead of any here.